src/Api.hs

```diff
@@ -1,15 +1,21 @@
-{-# LANGUAGE DataKinds     #-}
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE TypeOperators #-}
 
 module Api (app) where
 
-import           Control.Monad.Reader (runReaderT)
-import           Servant              ((:<|>) ((:<|>)), Proxy (Proxy), Raw,
-                                       Server, serve, serveDirectoryFileServer)
-import           Servant.Server
-
-import           Api.User             (UserAPI, userApi, userServer)
-import           Config               (AppT (..), Config (..))
+import Control.Monad.Reader (runReaderT)
+import Servant
+       ( (:<|>)((:<|>))
+       , Proxy(Proxy)
+       , Raw
+       , Server
+       , serve
+       , serveDirectoryFileServer
+       )
```

Comment:
If a newline indented import list expands beyond the column count, then it'll put each term on a new line. This takes up space, but it's really easy to read, and the diff for adding or removing an import line points to exactly the change that was made.